lib/util.js

```diff
@@ -222,3 +222,22 @@ exports.parseFormats = function(info, debug) {
   formats.sort(exports.sortFormats);
   return formats;
 };
+
+
+
+/**
+ * @param {String} videoDescriptionHtml
+ * @return {String}
+ */
+exports.cleanVideoDescription = function(videoDescriptionHtml) {
+	if (!videoDescriptionHtml || videoDescriptionHtml == null) 
+		return null;
+
+	return entities.decode(videoDescriptionHtml
+		.replace('\n', ' ', 'gi')
+		.replace(/\s*\<\s*br\s*\/?\s*>\s*/gi, '\n')
+		.replace(/<\s*\/\s*p\s*>\s*<\s*p[^>]*>/gi, '\n')
+		.replace(/<.*?>/gi, '')
```

Comment:
After examining video descriptions lines like:

```2  Song name  <a href="#" onclick="yt.www.watch.player.seekTo(5*60+42);return false;">5:42</a><br>```

I think that the reasoning after those two differentes regexes is:

1. First, replace 'br' and an open + closing 'p' html tags with newlines (like the one at the end of the example line)
2. Remove ANY other html tag that remains after the previous step.

I think it works pretty well. Where do you exactly see ```<[^>]*>``` fit in those replacements? Let me know and I'll try to research some more on that direction.